src/services/ebic/index.jsx

```
@@ -0,0 +1,306 @@
+
+  useEffect(() => {
+    if (!isTop) {
+      getAllLikes();
+      getAllComments();
+    }
+  }, [image?.gebc_id]);
+
+  return (
+    <ImagePreviewer
+      {...{
+        image: image?.image,
```

ESLint

[error] 'image.image' is missing in props validation

(react/prop-types)